### PR TITLE
LL-2985 (AccountPage): add bitcoin address tabs

### PR DESCRIPTION
**Pull request by LFBarreto**

### CODE REVIEW

File: src/renderer/families/bitcoin/AccountTabs.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const freshAddresses = account.freshAddresses || [];
+
+  const receiveAddress = useCallback(
+    (address, path) => {
+      dispatch(
+        openModal("MODAL_RECEIVE", {
+          account: { ...account, freshAddress: address, freshAddressPath: path },
```
Comment (LFBarreto) [inline src/renderer/families/bitcoin/AccountTabs.js:97]: @gre :wave: hi we need your input on this part of the code.
What we need to do is be able to trigger a receive flow from a specific bitcoin address. 
The common logic as of now doesn't allow that directly so either we do this little trick here in "emulating" an account with the address we want to receive on, or we update the receive logic so we can force these params maybe.
WDYT?

Comment (gre) [reply src/renderer/families/bitcoin/AccountTabs.js:97]: no we shouldn't do this. we need instead to pass a second parameter that tells the index to use.
that index needs to be passed all along. typically to the bridge.receive.
we must do this properly 🙏 

in scope of this, we must make an evolution on CLI to also debug it from there, etc.. i'm sorry to not have anticipated on that. let's try to write it down for next sprint.

File: src/renderer/families/bitcoin/AccountTabs.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          parentAccount,
+          skipAccountSelect: true,
+        }),
+      );
+    },
+    [openModal, dispatch, account, parentAccount],
+  );
+
+  console.log(freshAddresses);
```
Comment (gre) [inline src/renderer/families/bitcoin/AccountTabs.js:106]: console.log leaked

File: src/renderer/families/bitcoin/AccountTabs.js (hunk 1 above)
Comment (gre) [reply src/renderer/families/bitcoin/AccountTabs.js:97]: ok looks like it's an index in freshAddresses? so index? but what about old addresses tho. Didn't we wanted to list used addresses too?

**Review by gre — CHANGES_REQUESTED** [review]
let's discuss

**Comment by gre** [comment]
it seems this work is postponed so we'll close. work can always be continued from this history.

**Pull request closed without merging**